tests/integration_tests/test_hyperopt_ray.py

```diff
@@ -13,29 +13,37 @@
 # limitations under the License.
 # ==============================================================================
 import contextlib
+import json
 import logging
 import os.path
+import sys
+from typing import List
 
 import mlflow
 import pandas as pd
 import pytest
 from mlflow.tracking import MlflowClient
+from pyexpat import model
+from ray import tune
```

Comment:
Wondering if this import is needed and if it'll cause the same error that the callback was throwing - would be good to double check!

Reply:
Pre-commit CI bot added that too.